src/test/java/org/example/BaseTest.java

```diff
@@ -1,42 +1,30 @@
 package org.example;
 
-
-import com.networknt.schema.*;
 import org.testng.annotations.BeforeSuite;
 
-import java.util.List;
-import java.util.Locale;
 import java.util.Properties;
-import java.util.Set;
-import java.util.stream.Collectors;
 
 public abstract class BaseTest {
 
     Properties loadProperties;
 
+    HttpService httpService;
+
+    JsonValidator jsonValidator;
+
+    TestDataFactory testDataFactory;
+
     protected String token;
 
     @BeforeSuite
```

Comment:
@BeforeSuite runs once before all tests in the suite, so all test methods share the same instances.
Use @BeforeMethod, to initialize objects before each test method.